packages/grid/x-data-grid/src/hooks/features/rows/gridRowsSelector.ts

```diff
@@ -41,3 +41,19 @@ export const gridRowTreeDepthSelector = createSelector(
 );
 
 export const gridRowIdsSelector = createSelector(gridRowsStateSelector, (rows) => rows.ids);
+
+/**
+ * @ignore - do not document.
+ */
+export const gridAdditionalRowGroupsSelector = createSelector(
```

Comment:
It's just a side question but if we don't want to document specific selectors because they are private is it enough to just not document them or does it also make sense to add the `unstable_` prefix to them?

Reply:
We don't have any selector with `unstable_` prefix.
We can add the prefix or remove it from exports

Reply:
On aggregation I did not prefix them.
But that's a good question.

I would say that if the feature is under `experimentalFeature`, we don't have to prefix anything.

Reply:
I'm ok with the current approach. The addon is more when a user is using IntelliSense in their editor to get an indication that this thing is private/unstable.

Reply:
I would prefer to just not export this selector, at least for now.

Reply:
For the selector, if he does not have a real application in user land, no problem to wait before exporting it indeed

Reply:
I've removed row pinning selectors from exports